torch/csrc/autograd/functions/tensor.cpp

```diff
@@ -159,5 +176,38 @@ void CopySlices::release_variables() {
   fn = nullptr;
 }
 
+void CopySlices::compiled_args(CompiledNodeArgs& args) {
+  TORCH_CHECK(!view_fn, "view_fn not supported by compiled autograd")
+  TORCH_INTERNAL_ASSERT((bool)fn);
+  args.collect(base);
+  args.collect(view);
+  args.collect(fn);
```

Comment:
How do we should view_fn is ok here?

Reply:
There is an assert it is not set a few lines up.

Reply:
Oups, time to go home.
IIRC Dynamo force enables the config that view_fn must be used (instead of TensorGeometry) so we will most likely need to figure that one out.

Reply:
Yeah, though I'd expect dynamo produced graphs to show up as a single AotAutograd node in the backwards graph and not use this code path.

`view_fn` is pretty tricky to handle since it is a C++ lambda that very likely has SymInts hiding inside it.  My best idea for handling it is replacing it was a new class and virtual method.  Maybe there is some more elegant way I am not thinking of though...